driver-core/src/test/java/com/datastax/driver/core/policies/ErrorAwarePolicyTest.java

```diff
@@ -190,6 +226,7 @@ public void should_exclude_host_after_reaching_inclusion_threshold() throws Inte
 
         try {
             Session session = cluster.connect();
+            cluster.register(latencyTracker);
```

Comment:
Good idea but we have no guarantee that one of the trackers will be called before the other as they're stored in a Set. Just wondering if it matters here ?